src/backend/binaryen/lib/interface/meta.c

```diff
@@ -9,65 +9,69 @@
 
 
 /**
- * MetaField record the properties information of object
+ * MetaProperty record the properties information of object
  * name: property name
  * flag_and_index: flag and index of the property
  * type: type of the property, represent by property's type id
 */
-typedef struct MetaField {
+typedef struct MetaProperty {
     char *name;
     int flag_and_index;
     int type;
-} MetaField;
+} MetaProperty;
 
 /**
  * Meta record type id and the properties information of object
  * type_id: type id of the object
- * count: number of fields
- * fields: fields information
+ * count: number of properties
+ * properties: properties information
  */
-
 typedef struct Meta {
     int type_id;
     int impl_id;
     int count;
-    MetaField fields[0];
+    MetaProperty properties[0];
```

Comment:
What's the difference between fields and property

Reply:
Property contain field, method and accessor, to avoid misunderstanding, I used property instead of field.